packages/ibm-products/src/components/TagSet/TagSetOverflow.tsx

```diff
@@ -87,6 +91,7 @@ export const TagSetOverflow = React.forwardRef(
       // The component props, in alphabetical order (for consistency).
 
       allTagsModalSearchThreshold = defaults.allTagsModalSearchThreshold,
+      disablePopOver = false,
```

Comment:
same goes here, but that's just a suggestion, not mandatory change!

Reply:
See above.